src/Raven.Client/Http/RoundRobinNodeSelector.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Raven.Client.Http
+{
+    public class RoundRobinNodeSelector : FailoverNodeSelector
+    {
+        public RoundRobinNodeSelector(Topology topology) : base(topology)
+        {
+        }
+
+        public override void OnSucceededRequest()
```

Comment:
This is not a good idea. We want to keep the same node for the duration of a session, I think.
Otherwise you might load two docs from different nodes.